constructor/header.sh

```diff
@@ -213,6 +213,57 @@ then
     fi
 #endif
 
+#if ppc64le
+    if [ "$(uname -m)" != "ppc64le" ]; then
+        printf "WARNING:\\n"
+        printf "    Your machine hardware does not to appear be Power8 (little endian), \\n"
```

Comment:
Please change:

> does not to appear be

.. to:

> does not appear to be

(in all of these)

Reply:
Fixed in the commit below.